pkgs/games/heroic/fhsenv.nix

```diff
@@ -12,6 +12,7 @@ buildFHSUserEnv {
 
   targetPkgs = pkgs: with pkgs; [
     heroic-unwrapped
+    gamemode
```

Comment:
With `programs.gamemode.enable = true;`, isn't `gamemode` visible in the FHS even without it being in `targetPkgs`?

Reply:
It seems like it should, but without this, trying to launch games from Heroic with the GameMode option enabled fails with a `dlopen()` error.

Reply:
Ok, I was wondering this because I was working on the updating Bottles, which didn't require it (though Bottles itself isn't in an FHS environment, so that's probably the difference).